src/components/molecules/tooltip/tooltip.module.scss

```
@@ -0,0 +1,64 @@
+@use 'src/styles/variables' as vars;
+
+.tooltip {
+  font-family: vars.$yandexFont;
+  position: absolute;
+  bottom: 60px;
+  left: -60px;
+  z-index: 1000;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #000;
+  width: 226px;
+  padding: 4px 4px;
+  border: 5px solid #fff;
+  border-radius: 15px;
+  justify-content: center;
+  color: #fff;
+  background-color: #fff;
+  // background-color: #333;
+  text-align: center;
+  font-weight: 500;
+  font-size: vars.$font-size-m;
+  color: #212329;
+
+  &Row {
+    &Title {
+      border-top-right-radius: 5px;
+      border-top-left-radius: 5px;
+      display: flex;
+      padding: 10px;
+      min-height: 20px;
+      background-color: vars.$drop-shadow-small;
+      color: #212329;
+
+      &Img {
+        width: 30px;
+        height: 30px;
+        object-fit: cover;
+        font-size: 7px;
+        border: 1px solid #fff;
+        border-radius: 50%;
+        margin: 0 20px 0 0;
+      }
+      &Text {
+        margin: 5px;
+      }
+    }
+    &Text {
```

```suggestion
      }
      
      &Text {
        margin: 5px;
      }
    }
    
    &Text {
```
Селекторы разделяем пустой строкой